tests/regression/ba-issues/README.md

```diff
@@ -0,0 +1,250 @@
+            "description": ""
+        }
+    },
+    "runtime": "iwasm-multi-tier-wasi-disabled",
+    "file": "out.aot",
+    "mode": "aot",
+    "options": " --heap-size=0 -f to_test",
+    "argument": "",
+    "expected return": {
+        "ret code": 0,
+        "stdout content": "0x0:i32",
+        "description": "expected output 0x0:i32"
+    }
+}
+```
+
+### For deprecated issue test cases
+
+Due to the spec changes, some cases may be deprecated in the future. When the running result is not as expected and after making sure it's not the bug in WAMR but the test cases should be deprecated(maybe using wasm-validate or some other tool). They should be moved into directory `issues-deprecated.` And simply set the `"deprecated": true,` in their corresponding running configuration.
+
+For example:
+
+```JSon
+{
+    "deprecated": true,
+    "ids": [
+        47,
+        48,
+        49,
+        50,
+        51,
+        52,
+        53,
+        54,
+        55,
+        56,
+        57,
+        58,
+        59,
+        60,
+        61,
+        62,
+        63,
+        64,
+        65,
+        66,
+        67,
+        68,
+        69,
+        70,
+        71,
+        72,
+        73,
+        74,
+        75,
+        76,
+        77,
+        78,
+        79,
+        80,
+        81,
+        82,
+        83,
+        84
+    ],
+    "runtime": "iwasm-default",
+    "mode": "classic-interp",
+    "file": "PoC.wasm",
+    "argument": "",
+    "expected return": {
+        "ret code": 0,
+        "stdout content": "",
+        "description": "no segfault"
+    }
+}
+```
+
+## Running test cases and getting results
+
+simply run `run.py`
+
+```shell
+./run.py
+```
+
+If everything went well, you should see similarly output in your command line output
+
+```shell
+Finish testing, 22/22 of test cases passed, no more issues should further test
+```
+
+If you add the test case under directory `issues` but forget to add the running config in json file, the output can be something like
+
+```shell
+Finish testing, 21/21 of test cases passed, {2945} issue(s) should further test
+```
+
+If some test case are failing, then it will be something like
+
+```shell
+Finish testing, 21/22 of test cases passed, no more issue(s) should further test
+```
+
+And a log file named `issues_tests.log` will be generated and inside it will display the details of the failing cases, for example:
+
+```dummy
+=======================================================
+Failing issue id: 2945.
+run with command_lists: ['./build/iwasm-default-wasi-disabled', '--heap-size=0', '-f', 'to_test', '/home/tl/TL/clion_projects/wamr-internal-test/tests/regression/ba-issues/issues/issue-2945/iwasm_fast_interp_moob_unhandled.wasm']
+exit code (actual, expected) : (1, 0)
+stdout (actual, expected) : ('Exception: out of bounds memory access', 'Exception: out of bounds memory access')
+=======================================================
+```
+
+TODO: only run part of test cases?
```

Comment:
How about removing the this line? We may implement it in the future but seems no need to add it in the document?

Reply:
OK. I also think it is better to remove it.